dashboard/app/controllers/pd/workshop_user_management_controller.rb

```diff
@@ -1,35 +1,68 @@
 class Pd::WorkshopUserManagementController < ApplicationController
   authorize_resource class: :pd_workshop_user_management
 
+  # restrict the PII returned by the controller to the view by selecting only these columns from the model
+  RESTRICTED_USER_ATTRIBUTES_FOR_VIEW = %w(
+    users.id
+    email
+    name
+    user_type
+    current_sign_in_at
+    sign_in_count
+    users.created_at
+  ).freeze
+
   # get /pd/workshop_user_management/facilitator_courses
   def facilitator_courses_form
     search_term = params[:search_term]
     if search_term =~ /^\d+$/
-      @facilitator = User.joins(:permissions).find_by(
-        id: search_term,
-        user_permissions: {permission: UserPermission::FACILITATOR}
-      )
+      @user = restricted_users.find_by(id: search_term)
     elsif search_term
-      @facilitator = User.joins(:permissions).find_by(
-        hashed_email: User.hash_email(search_term),
-        user_permissions: {permission: UserPermission::FACILITATOR}
-      )
+      @user = restricted_users.find_by(hashed_email: restricted_users.hash_email(search_term))
     end
 
-    unless @facilitator || search_term.blank?
-      flash[:notice] = "Facilitator not found"
+    unless @user || search_term.blank?
+      flash[:notice] = "User not found"
     end
   end
 
   # post /pd/workshop_user_management/assign_course
   def assign_course
```

Comment:
Be more verbose in the method name - you are assigning a course for a facilitator and the name should reflect that